source/channels/make-calls.rst

```diff
@@ -16,30 +17,44 @@ Mattermost introduces beta voice calling and screen sharing functionality in cha
 
 - `Debugging <#debugging>`__
 
-Features available in Mattermost 7.0
-------------------------------------
+Features 
+--------
 
-From 7.0, you can:
+Starting from Mattermost v7.0, you can:
 
 - Start or join a call from desktop and mobile apps in addition to web browsers.
 - Share screen (not available on mobile), raise hands, chat in a thread, and even switch between products in the Mattermost suite (i.e., Channels, Boards, and Playbooks) while on a call.
 - Start a call using the ``/call start`` slash command or **Start call** in the channel header.
 
-Mattermost Cloud users can start calling right out of the box. For Mattermost self-hosted deployments, System Admins need to enable the plugin and adjust configurations `in the System Console </configure/configuration-settings.html#calls-beta>`_.
+Starting from Mattermost v7.7 you can:
 
-To start a call, select **Start call** in the channel header. Any active team member in the channel can join a call, whether it's a public or private channel. If someone from outside of the organization wants to join a call, you'll need to provide them with a guest account, and add them to the channel. Users who are archived or not registered won't be able to join a call.
+- Record calls (Mattermost Enterprise only).
+- React using emojis.
+
+Mattermost Cloud users can start calling right out of the box. For Mattermost self-hosted deployments, System Admins need to enable the plugin and adjust configurations `in the System Console </configure/configuration-settings.html#calls>`_.
```

Comment:
Link is still broken, it should point to `configure/plugins-configuration-settings.html#calls`.